lib/refile/app.rb

```diff
@@ -85,6 +85,16 @@ class App < Sinatra::Base
       "forbidden"
     end
 
+    error ArgumentError do
```

Comment:
@jnicklas should it raise different kind of exception (ie Refile::InvalidFile or UnreadableFile) in backend macros?